### DISCUSSION TITLE
api definition

### DISCUSSION THREAD

**Discussion by epogrebnyak** [cncf/glossary#132] [General]
Is discussion of existing terms appropriate here? In API article you have: 

> As applications become more complex, small code changes can have drastic effects on other functionality. Applications need to take a modular approach to their functionality if they can grow and maintain stability simultaneously. 

Having an API tell you nothing about modularity of application. A monolith can perfectly have a good API. API provides a contract and encapsulates what is inside, tell nothing of how that inside is structured.

**Comment by SaxenaAnushka102** [comment]
Hi @epogrebnyak ! That looks like a point worth discussing. You can raise an issue for the same. Please refer to the [Contributing guideline](https://github.com/cncf/glossary/blob/main/CONTRIBUTING.md).
